### PR TITLE
Comment update

### PR DESCRIPTION
**Pull request by tingkai-zhang**
The `param targets` fed to classifier is the target label after being fit_transform(). The shape of `param targets` should be [batch size, 2]
For example, we have classification task with 3 categories: 0,1,2.  
The list of labels [0, 1, 2] will be transformed to [array([1, 0]), array([0.5, 0.5]), array([0, 1])]
I am not sure whether I misunderstand but `n_classes` here may not refer to number of classes?

### CODE REVIEW

**Comment by benleetownsend** [comment]
The encoding used for the normal classification targets is a 1-hot representation, therefore the n_classes argument is equal to the number of unique classes to predict. 

There is some code floating arround in the repo for a model that does “ordinal classification” whereby the encoding format you have described is used. In this unique case, n_classes is hard coded to 2 and does not truely reflect the documentation. I think this could be better documented but I think as typically it is equal to the number of classes, a hybrid of the previous documentation and your suggested change would be most appropriate.

**Comment by madisonmay** [comment]
Closing as this has been resolved in a separate PR. Thanks for the catch!
